Marlin/Marlin_main.cpp

```diff
@@ -1579,7 +1647,7 @@ static void setup_for_endstop_move() {
       feedrate = homing_feedrate[Z_AXIS];
 
       // Move down until the Z probe (or endstop?) is triggered
-      float zPosition = -(Z_MAX_LENGTH + 10);
+      float zPosition = MIN_PROBE_PT;
```

Comment:
@thinkyhead this movement is still relative to the origin. Therefore, `-(Z_MAX_LENGTH + 10)` would be a huge negative number that is essentially guaranteed to be outside of the range of the Z axis (`Z_MIN_POS` to `Z_MAX_POS`).

`MIN_PROBE_PT` is set in Configuration.h and could easily be set to any value, including `-(Z_MAX_LENGTH + 10)` if desired for any reason.

Why exactly do you think the position is lost during probing? `set_bed_level_equation_lsq(plane_equation_coefficients)` is not called until after probing is completed. Maybe the position isn't well-known if you are using the same mechanism for probing as you are for homing, in which case you can still simply change the value of `MIN_PROBE_PT` in Configuration.h.